src/components/WorldMap.js

```diff
@@ -153,12 +161,25 @@ export class WorldMap extends Component {
             localStorage.setItem('geolocation', 'true');
             const { latitude, longitude } = location.coords;
             this.state.map.setView([latitude, longitude]);
+            locationSet='true';
           });
         } else {
           localStorage.removeItem('geolocation');
         }
       });
     }
+
+    if(locationSet='false'){
+      let url=window.location.href;
+      for (let country in domainCoors){
+        if(url.indexOf(country) != -1){
```

Comment:
this will match a lot of false positive, for instance `https://www.lockdown.world/europe` or  `https://www.lockdown.world/foo?europe`